src/cli/cbenv_register.rs

```diff
@@ -207,42 +205,28 @@ fn clusters_register(
     ))
 }
 
-pub fn update_config_file(guard: &mut MutexGuard<State>, span: Span) -> Result<(), ShellError> {
-    let path = match guard.config_path() {
-        Some(p) => p,
-        None => {
-            return Err(generic_error(
-                "A config path must be discoverable to save config",
-                None,
-                span,
-            ));
-        }
-    };
-    let mut cluster_configs = Vec::new();
-    for (identifier, cluster) in guard.clusters() {
-        cluster_configs.push(ClusterConfig::from((identifier.clone(), cluster)))
-    }
-    let mut capella_configs = Vec::new();
-    for (identifier, c) in guard.capella_orgs() {
```

Comment:
Why have we removed this?

Reply:
The Capella orgs get read and written back to the file, so we don't need to do this. With this approach the API keys always get written to the config, which we don't want if they were given in the credentials file.